modules/doc/content/framework_development/sanity_checking.md

```diff
@@ -33,6 +33,18 @@ MOOSE performs several sanity checks just before the simulation runs. This inclu
 - All pluggable systems must refer to the right type of variable (scalar, vector, array, "nonlinear", "auxiliary", etc)
 - Material properties must be defined on every block or boundary where they are consumed.
 - Multiple material properties with the same name be defined on overlapping blocks/boundaries.
+- Variable definition on boundary restricted objects. Nodal object integrity
+  checking is controlled with
```

Comment:
integrity or coverage?
we call it coverage for variables and materials right

Reply:
that is the parameter name for those. But there is also `fv_bcs_integrity_check`. And then all these coverage/integrity checks occur through the `CheckIntegrityAction` and `FEProblemBase::checkProblemIntegrity`

Reply:
kk I dont mind either way. This parameter is not to be used by many users anyway